registries_conf_ctl/cli.py

```diff
@@ -166,50 +166,55 @@ def dump(self):
 
 class RegistriesConfV2(Fmt):
 
-    def __init__(self, f):
+    def __init__(self, f, allow_empty_config=False):
```

Comment:
registries_conf_ctl/cli.py:169: error: Type signature has too few arguments
registries_conf_ctl/cli.py:169: error: Function is missing a type annotation